README.md

```diff
@@ -865,10 +867,29 @@ It's important to ensure that `LD_LIBRARY_PATH` includes `/usr/local/lib` and `/
 
 #### FreeBSD
 
-FreeBSD uses pkg as the package manager.  You should install the dependency expat.  libz is already installed.  As with NetBSD, you should use pkg to install python3, cmake, bash, sudo, gettext and gcc.  The default GCC compiler is currently 8.3.0.
+Clang is pre-installed as ``/usr/bin/{cc|c++}` as well has libz and expat.  FreeBSD uses pkg as the package manager which I used to install cmake and git.
+
+```bash
+$ su root
+Password:
+# pkg install cmake
+# pkg install git
+```
+
+To run the Exiv2 test suite, I installed bash and python.  The test suite requires additional work as the platform `diff` command does not understand the option `--binary` and returns an error.  In consequence, the test harness returns lots of errors.  I hope to address this in v0.27.3.
```

Comment:
Do we have an issue in github for this task?

Reply:
Yes.  This is discussed in #902 which I will reopen for v0.27.3